app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt

```diff
@@ -2312,6 +2312,38 @@ class ChatController(args: Bundle) :
                         clipboardManager.setPrimaryClip(clipData)
                         true
                     }
+                    R.id.action_mark_as_unread -> {
+                        val chatMessage = message as ChatMessage?
+                        ncApi!!.setChatReadMarker(
+                            credentials,
+                            ApiUtils.getUrlForSetChatReadMarker(
+                                1,
+                                conversationUser?.baseUrl,
+                                roomToken
+                            ),
+                            chatMessage!!.jsonMessageId.minus(1)
```

Comment:
@nickvergessen sorry forgot to ask, I don't 'know' the message id of the message above the one where you click 'mark as unread' but simply do an 'id-1'.. Is that okay? Seems to work :D

Reply:
Will break when it's in another conversation. What the webUI does it when creating the message list add the previous message id to this message element as "previousId".
But I also start seeing the API to be not very good here. It comes from the fact that we just have one "mark read here" endpoint. Maybe a new one should be added which then takes care of all this.

Reply:
I'll have a look, but thatz one might be tricky, let's see

Reply:
So be fixed now :)